requirements.txt

```diff
@@ -1,36 +1,35 @@
-atomicwrites==1.3.0
-attrs==19.1.0
-base58>=1.0.3
-certifi>=2019.6.16
-chardet>=3.0.4
-decorator==4.4.0
-dogpile.cache==0.7.1
-docker==4.2.0
-falcon==2.0.0
-greenlet==0.4.15
-gunicorn==19.9.0
-idna>=2.8
-importlib-metadata==0.19
-jsonschema==2.6.0
-Mako==1.0.14
-MarkupSafe==1.1.1
-meinheld==1.0.1
-more-itertools==6.0.0
-pluggy==0.12.0
-protobuf==3.9.1
-py==1.10.0
-pytest==4.3.1
-python-dateutil==2.8.0
-python-editor==1.0.4
-python-mimeparse==1.6.0
-pytz==2018.9
-redis==2.10.6
-requests>=2.22.0
-requests-mock==1.7.0
-urllib3==1.26.5
-xxhash==1.3.0
-zipp==0.5.2
+atomicwrites>=1.4.1
```

Comment:
@RonaldCrb You locally tested and verified everything works?

Reply:
yes
![Screenshot 2024-10-07 at 3 15 10 PM](https://github.com/user-attachments/assets/dfbb12b9-acca-4b56-bd00-107fcb8a6895)

Reply:
@roccomuso what do you think? can we progress with this now? this is blocking us from applying updates to polkadot

Reply:
i see a bunch of major version updates. were there no breaking changes?

has this been deployed to dev and tested?

Reply:
I agree, so many major upgrades. Is this mandatory?
I'm not familiar with polkadot backend, can't help much